kge/config.py

```diff
@@ -487,7 +487,7 @@ def last_checkpoint_number(self) -> Optional[int]:
             tried_epoch += 1
             if os.path.exists(self.checkpoint_file(tried_epoch)):
                 found_epoch = tried_epoch
-        if found_epoch > 0:
+        if found_epoch >= 0:
```

Comment:
This doesn't work as it never even tries 0 above; the init values need to be updated as well. Also, 500 may not be enough anymore for some models...

Reply:
actually did work, but did not check if it actually existed. 
Improved this, so that we don't iterate over every number until 500